### DISCUSSION TITLE
How to Setup and Calibrate

### DISCUSSION THREAD

**Discussion by KaHu85** [ClassicDIY/SkyeTracker#45] [General]
Hello Graham,

I am from Austria and have been rebuilding a similar solar tracker for the last two days.
Today I have connected the first axis and it is already moving.

Now my question.
What is the best way to calibrate the whole thing?

I thought I set after a compass on the phone times roughly the limits for Max East and West.
The length of the actuator and the speed.

Unfortunately, the sun was already gone and the panel is now between sunrise and sunset.

Now I am curious about tomorrow.

 
One more question:
I have for the tilt a door drive, unfortunately, this only goes after 5 seconds. Could I still use that if I reprogram it somewhere?
When closing it starts a little faster, but even here I would probably need a second until he then goes.

I would have to know at which point I would have to reprogram this, as well as the speed and inclination of this axis...

Thanks for this great project!
Will give you another beer or coffee for it then :)

Here you See the Tracker: https://berkmann.cloud/s/DLsdPTxjgLmPsr4

Greetings Alex

**Comment by KaHu85** [comment]
Okay, my fears have come true. The panel stood still today. When I got home I noticed that the latitude and longitude are set to completely wrong values. There is no way to change them manually in the APP although I can get the coordinates with another APP. Have it then bypassed in that I have the ESP32 directly with the values under: Configuration.h line 59 and 60 manually. double getLat() { return 47.xxx628; } and double getLon() { return 9.xxx297; }
Now it shows the data like this in the Android APP under Info.
The tracker still drove its way this evening.
Now I am curious about tomorrow.

**Comment by KaHu85** [comment]
So today the tracker worked perfectly! Maybe you can still answer the input comment on occasion and perhaps address the location issue why it does not work in the Android app. Thanks LG Alex

**Comment by graham22** [comment]
Ok, I'll assemble a new test setup in the next couple of days...

On Tue, Sep 26, 2023 at 11:00 AM KaHu85 ***@***.***> wrote:

> So today the tracker worked perfectly! Maybe you can still answer the
> input comment on occasion and perhaps address the location issue why it
> does not work in the Android app. Thanks LG Alex
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClassicDIY/SkyeTracker/discussions/45#discussioncomment-7113970>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIYATP2Q6DKJT3JD4NVEXTX4LUXVANCNFSM6AAAAAA5FDL2RE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
Graham A. Ross
(613) 769-1331

**Reply by KaHu85** [reply]
Hey Graham, any update?

**Comment by graham22** [comment]
looking at it, I first have to upgrade the google api for the app

On Mon, Jan 22, 2024 at 5:46 PM KaHu85 ***@***.***> wrote:

> Hey Graham22 any update?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClassicDIY/SkyeTracker/discussions/45#discussioncomment-8214114>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIYATLNL2C3XNHHDRO7F5LYP3T43AVCNFSM6AAAAAA5FDL2RGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DEMJUGEYTI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Graham A. Ross
(613) 769-1331
